lib/docker/executeDockerBuild.ts

```diff
@@ -100,11 +125,10 @@ export interface DockerOptions {
 export type DockerImageNameCreator = (p: GitProject,
                                       sdmGoal: SdmGoalEvent,
                                       options: DockerOptions,
-                                      ctx: HandlerContext) => Promise<{ registry: string, name: string, tags: string[] }>;
+                                      ctx: HandlerContext) => Promise<{ name: string, tags: string[] }>;
```

Comment:
I think the ImageNameCreator should just take the list of registries from the configuration options and return all desired registry/name/tag combinations. That preserves greater flexibility to decide where we want to push to.

So change the return type to `Array<{ registry?: string, name: string, tag?: string }>`

Reply:
Will update; as discussed.